### DISCUSSION TITLE
Benefit to using rgb24 instead of yuv420p?

### DISCUSSION THREAD

**Discussion by covermeyer24** [nagadomi/nunif#219] [Q&A] [answered]
If the source file I'm converting is using yuv420p is there any difference in quality using rgb24 instead of yuv420p when converting?

**Comment by nagadomi** [comment, marked as answer]
For iw3, there is almost no quality difference unless the output format is Anaglyph.
Personally, I do not recommend rgb24 because of larger file size and lower compatibility.
